test/fluree/db/query/misc_queries_test.clj

```diff
@@ -32,26 +32,26 @@
                                 :where  [['?s :type :ex/User]]}))))))
 
 (deftest ^:integration s+p+o-full-db-queries
```

Comment:
The name `s+p+o-full-db-queries` doesn't make sense for this test anymore. We might want to rename it or create a different `deftest` for these opst tests